server/resources/resource_management.go

```diff
@@ -316,6 +316,23 @@ func GetMultipleResources(context middleware.Context, dataStore db.DB, resourceS
 	currCond := policy.GetCurrentResourceCondition()
 
 	filter := FilterFromQueryParameter(resourceSchema, queryParameters)
+	if search_filter,ok := queryParameters["search_field"]; ok && resourceSchema.IsSubstringSeachEnabled(){
+			for _,column := range search_filter{
+				if search_value,ok := queryParameters[column]; ok {
+					delete(filter, column)
+					searchValue := strings.Builder{}
+					searchValue.WriteString("~")
```

Comment:
I think tweaking original input with magic prefix tilde is not good idea. It makes hard for other developers to understand how it works, and may occur some unexpected errors (e.g., users may use tilde as name field, description field and so on...). 

We should manage what properties are used for substr match in other fields. Actually, I like @michalzurawski 's idea, which is discussed offline, that enhance `paginator` below.

Reply:
+1 about `~` being problematic. Can you guys paste Michal's idea here?